test/kubemark/iks/util.sh

```diff
@@ -106,15 +109,16 @@ function set-registry-secrets {
 function set-hollow-master {
 	echo -e "${color_yellow}CONFIGURING MASTER${color_norm}"
 	master-config
-	MASTER_IP=$(cat $KUBECONFIG | grep server | awk -F "/" '{print $3}')
+	# shellcheck disable=2034 # MASTER_IP used in parent script
+	MASTER_IP=$(grep server "$KUBECONFIG" | awk -F "/" '{print $3}')
```

Comment:
Should this instead be exported, and the function commented as such?  I think this is how I've seen this handled elsewhere

Reply:
I think it'd be okay, if so there are 2 others that should also be exported. `KUBECTL` and `TEST_CLUSTER_API_CONTENT_TYPE`. Both are set within a function in the util script, but used elsewhere.

Reply:
Yeah that sounds good

Reply:
Should be good. :)

Reply:
still needs the comment I think?

Reply:
They do -_- I completely forgot to add that part. Will resolve shortly.